models.go

```diff
@@ -0,0 +1,14 @@
+package leaseweb
+
+// Structs for easy json marshalling.
+// Only declare fields that are used.
+type LeasewebRecordSet struct {
+	Name    string   `json:"name"`
+	Type    string   `json:"type"`
+	Content []string `json:"content"`
+	TTL     int      `json:"ttl"`
+}
+
+type LeasewebRecordSets struct {
+	ResourceRecordSets []LeasewebRecordSet `json:"resourceRecordSets"`
+}
```

Comment:
FYI, if these aren't used outside of this package, you can un-export them (start with lowercase, like `leasewebRecordSet`).

Reply:
Thanks! I'll fix this in the repo.

Reply:
Cool; same goes for any identifier that isn't needed publicly. General rule is, if it's only used within the package, keep the type name lowercase to avoid increasing your exported API surface. :+1:

Reply:
Makes sense! New to go so still need to get used to the syntax. I'll also look into the Caddy module, I was planning to check Caddy out sometime anyway 👍🏻

Reply:
Neat. You might be the first libdns consumer that isn't using Caddy already -- what do you use libdns for, being new to Go and such?

Reply:
Oh, now that you mention it. I was confused with Traefik. Yes I started this implementation for Caddy. Oops.

I'm looking into building caddy with my provider with xcaddy as we speak 😄